Gemfile

```
@@ -16,7 +16,7 @@ gem "hashdiff", "~> 1.0"
 gem "image_processing", "~> 1"
 gem "interactor", "~> 3"
 gem "kaminari", "~> 1"
-gem "notifications-ruby-client", "~> 5.1"
+gem "mail-notify"
```

```suggestion
gem "mail-notify", "~> 1"
```
Could you pin this to version 1 for consistency with other gems pinning?